product-service/handlers/catalogBatchProcess.ts

```diff
@@ -0,0 +1,79 @@
+import { SQSEvent } from 'aws-lambda';
+import { Client } from 'pg';
+import { SNS } from 'aws-sdk';
+
+const catalogBatchProcess = async (
+  event: SQSEvent
+): Promise<{ statusCode: number }> => {
+  const sns = new SNS({ region: 'eu-west-1' });
+  const client = new Client({ ssl: { rejectUnauthorized: false } });
+  client.connect();
+
+  try {
+    await client.query('BEGIN');
+    let maxProductPrice = -1;
+
+    for (const record of event.Records) {
+      const { title, description, price, count } = JSON.parse(record.body);
+      const priceAsNum = +price;
+      const countAsNum = +count;
```

Comment:
You can convert these properties to numbers in the csv parser: https://github.com/mafintosh/csv-parser/issues/49#issuecomment-414070999